tooling-server-configuration/src/main/java/org/jboss/eap/qe/microprofile/tooling/server/configuration/ConfigUtils.java

```diff
@@ -0,0 +1,127 @@
+package org.jboss.eap.qe.microprofile.tooling.server.configuration;
+
+import org.jboss.arquillian.config.descriptor.api.ArquillianDescriptor;
+import org.jboss.arquillian.config.descriptor.api.ContainerDef;
+import org.jboss.arquillian.core.api.annotation.Observes;
+import org.jboss.arquillian.test.spi.event.suite.BeforeSuite;
+
+import java.util.Optional;
+
+/**
+ * Provides access to configuration properties
+ */
+public class ConfigUtils {
```

Comment:
IMHO this name is too general. I would suggest something like `ArquillianContainerProperties`.

Reply:
Done